README.md

```diff
@@ -1,7 +1,7 @@
-# codewind-odo-extension
-Extension to Codewind providing support for odo projects: https://codewind.dev
+# Codewind ODO extension
+Extension to Codewind providing support for ODO projects: https://codewind.dev
 
-Add additional rules to support ODO extension
+## Add additional rules to support Codewind ODO extension
 ODO extension needs add additional rules for accessing OpenShift resources
 Steps to add additional rules:
 1. In your home directory, run the following command to clone the ODO extension repository
```

Comment:
In your home directory, run the following commands to clone the ODO extension repository: